docs/install_linux_mac.rst

```diff
@@ -183,8 +184,8 @@ Now install the pre-requirements with zypper:
 
 .. code-block:: none
 
-    sudo zypper -n install opt-python38 opt-python38-setuptools git-core java-11-openjdk-headless
-    sudo zypper -n install -t pattern devel_basis
+    sudo zypper -n install opt-python38 opt-python38-setuptools git-core java-11-openjdk-headless nano
+    sudo zypper -n install -t pattern devel_basis 
```

Comment:
You accidentally added some trailing whitespace in here which needs to be removed:
```suggestion
    sudo zypper -n install -t pattern devel_basis
```